sympy/series/limits.py

```diff
@@ -214,21 +215,41 @@ def doit(self, **hints):
         elif str(dir) == "-":
             cdir = -1
 
+        if e.has(Abs):
+            term = e
+            while term:
+                cnt = 0
+                for t in term.args:
+                    if t.has(Abs):
+                        cnt += 1
+                        break
+                if cnt == 1:
+                    term = t
+                else:
+                    fin = term
+                    term = 0
+            coeff = limit(fin.args[0], z, z0)
```

Comment:
This limit does not seem to appear in the role of a coefficient. Maybe a better name could be found.